releasenotes/notes/deprecate-tracer-tags-attribute-e1b3706820110a09.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    :py:attr:`ddtrace.Tracer.tags` is deprecated. Use ``ddtrace.config.tags`` or the :ref:`DD_TAGS<dd-tags>` environment variable to set the global tags instead.
```

Comment:
`ddtrace.config.tags` won't work since we copy them onto the tracer :(

so the env variable is the only option, unless we get rid of `Tracer.tags` entirely and just use `ddtrace.config.tags` ?

Reply:
Changed this to just the env var. I see any refactoring of `Tracer._tags` outside the scope of 1.0.

Reply:
Yeah this is kinda tricky since there isn't really a replacement API unless we introduce one... which means we need to decide how we expect `tracer.tags` to work.